src/st/st_debug.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+extern ObjInit g_eBackgroundBlockInit[];
+extern u16 g_eInitGeneric2[];
+
+void EntityBackgroundBlock(Entity* self) {
```

Comment:
EntityBackgroundBlock is considered to be debug related?

Reply:
Perhaps we need to rename the files or functions. All I know is WRP had "st_debug.c", with an unnamed function, and I found that same function called EntityBackgroundBlock in CEN, in an unnamed file. Now the combination of the two results in EntityBackgroundBlock being in st_debug.h. Let me try seeing if the files can be shuffled around.